modules/ROOT/pages/import-api-specification-local-file.adoc

```diff
@@ -0,0 +1,37 @@
+= Importing an API Specification from a Local File
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API specification from your local machine and import it either into a new Mule Project, or an existing one.
+
+This task only works for RAML types of API specifications.
+
+. In the top task bar, select *File*, then select *New*, and then *Mule Project*.
+. Type a name for your project.
+. Select a Mule runtime version.
+. In the *API Specification* section, select the *From Design Center or Local Files* tab.
+. In the *Location* field, type the location of your local API, or click the *...* button and select *Browse Files...*.
+You can import a RAML file, or an entire zip file bundle if your API has schemas and examples.
+. Click *Finish*.
+
+Studio scaffolds your API specification. If you choose not to create flows out of your API specification, uncheck the *Scaffold Flows From These API Specifications* option.
```

Comment:
This task topic is titled "Importing," so is it confusing to use "scaffolds" as the result of this task?
Also, at what point does the user unselect the Scaffold Flows checkbox? 
Should this step be in the previous task?

If this option is really a checkbox...:

```suggestion
Studio scaffolds your API specification. If you choose not to create flows out of your API specification, unselect the *Scaffold Flows From These API Specifications* checkbox.
```
THIS IS A GLOBAL EDIT to each topic that contains this text.